Tests/scripts/slack_notifier.sh

```diff
@@ -9,6 +9,10 @@ echo "start slack notifier"
 # $2 = env_results_file_name
 # $3 = Slack channel
 
-python3 ./Tests/scripts/slack_notifier.py -n $IS_NIGHTLY -u "$CI_JOB_URL" -b "$CI_BUILD_ID" -s "$SLACK_TOKEN" -c "$CIRCLECI_TOKEN" -t "$1" -f "$2" -bu $IS_BUCKET_UPLOAD -j "$CI_JOB_NAME" -ca $ARTIFACTS_FOLDER -ch "$3"
+if [ "$CIRCLECI" != "true" ]; then
+  python3 ./Tests/scripts/slack_notifier.py -n $IS_NIGHTLY -u "$CI_PIPELINE_URL" -b "$CI_PIPELINE_ID" -s "$SLACK_TOKEN" -c "$GITLAB_STATUS_TOKEN" -t "$1" -f "$2" -bu $IS_BUCKET_UPLOAD -j "$CI_JOB_NAME" -ca "$ARTIFACTS_FOLDER" -ch "$3" -g "$CI_SERVER_URL" -gp "$CI_PROJECT_ID"
+else
+  python3 ./Tests/scripts/slack_notifier.py -n $IS_NIGHTLY -u "$CIRCLE_BUILD_URL" -b "$CIRCLE_BUILD_NUM" -s "$SLACK_TOKEN" -c "$CIRCLECI_TOKEN" -t "$1" -f "$2" -bu $IS_BUCKET_UPLOAD -j "$CIRCLE_JOB" -ca $CIRCLE_ARTIFACTS -ch "$3"
+fi
```

Comment:
Haven't we unified some of the environment variable names?

Reply:
yes but we still need to pass a few more details in case we are running from gitlab